doc/api/cli.md

```diff
@@ -626,6 +626,10 @@ added:
   - v13.3.0
   - v12.16.0
 changes:
+  - version: REPLACEME
```

Comment:
The man page probably needs to be updated as well.

Reply:
@cjihrig do you mean wasi.md or something else. I don't see any mention of the flag in wasi.md.

Reply:
I was referring to https://github.com/nodejs/node/blob/3cf65bd2688d35152accaca4a07df4ec603bdf9e/doc/node.1#L196-L197.

Reply:
The man page is `doc/node.1`

Reply:
@cjihrig removed.

Reply:
Updated.